Gemfile

```diff
@@ -12,6 +12,7 @@ group :test do
   gem 'bcrypt'
   gem 'rspec-expectations'
   gem 'rspec-mocks'
+  gem 'timecop'
```

Comment:
I would prefer if we didn't have to add extra test dependencies. The way this is used in specs seems simple enough that we could just mock the expectations.